lib/swoosh/adapters/postmark.ex

```diff
@@ -8,8 +8,9 @@ defmodule Swoosh.Adapters.Postmark do
 
       # config/config.exs
       config :sample, Sample.Mailer,
-        adapter: Swoosh.Adapters.Postmark,
-        api_key: "my-api-key"
+        adapter:  Swoosh.Adapters.Postmark,
+        api_key:  "my-api-key",
+        template: true # optional
```

Comment:
Not really, we should pick a different URL based on that: https://github.com/KamilLelonek/swoosh/blob/handle-templates-for-postmark/lib/swoosh/adapters/postmark.ex#L57-L61